### ISSUE TITLE
DE2 issue

### ISSUE THREAD

**Issue by rweigel** [hapi-server/server-bugs#9]
The csv response does not have the JSON prefixed by `#`:

https://hapi-server.org/servers/#server=CDAWeb&dataset=DE2_AC500MS_VEFI&parameters=orbitNumber&start=2013-01-28T13:25:51Z&stop=2013-01-30T13:25:51.000Z&return=data&format=csv&style=noheader

**Comment by rweigel** [comment]
Also, https://hapi-server.org/servers/#server=CDAWeb&dataset=HELIOS2_40SEC_MAG-PLASMA&parameters=ESS_Ang&start=2013-01-28T13:25:51Z&stop=2013-01-30T13:25:51.000Z&return=image&format=svg

**Comment by berniegsfc** [comment]
I can't reproduce the stated problem.  If I change the given example to style=header, I see the JSON prefixed with #.  And in both cases, I get no data 
```
"status": {"code": 1201, "message": "OK - no data for time range"}
```

**Comment by rweigel** [comment]
With noheader and no data, the response body should be empty. That is, the default of no header

```
curl "https://cdaweb.gsfc.nasa.gov/hapi/data?id=HELIOS2_40SEC_MAG-PLASMA&parameters=ESS_Ang&time.min=2013-01-28T13:25:51Z&time.max=2013-01-30T13:25:51.000Z" 
```

should return an empty body and ideally, and if possible, the usual 

```
HTTP/1.1 200 OK
```

modified to be

```
HTTP/1.1 200 OK HAPI 1201 - no data in the interval
```

**Comment by berniegsfc** [comment]
This is the same as [issue 18](https://git.mysmce.com/spdf/hapi-nand/-/issues/18).

**Comment by jbfaden** [comment]
I agree this is a bug.  I had to reread the documentation, because I thought it was okay to have a block like this is the data response and it's always bugged me that the mime type could change.  However, my reading of the 2.0 spec this morning is that this should be empty: https://github.com/hapi-server/data-specification/blob/master/hapi-2.0.0/HAPI-data-access-spec-2.0.0.md#hapi-status-codes last sentence: "If no JSON header was requested, then the HTTP error will be the only indicator of a problem. Similarly, for the data endpoint, clients may request data with no JSON header, and in this case, the HTTP status is the only place a client can determine the response status."
